xbmc/PlayListPlayer.h

```diff
@@ -103,6 +103,11 @@ class CPlayListPlayer : public IMsgTargetCallback
    */
   int GetCurrentPlaylist() const;
 
+  /*! \brief Get currently active playlist or a default for current player
```

Comment:
Maybe also changing the method name to `GetPreferredPlaylist(int default)` would be better?